gui.py

```diff
@@ -58,7 +77,7 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
     <div class="navbar-end">
       <div class="navbar-item">
         <div class="buttons">
-          <a href="https://uaf-iarc.typeform.com/to/mN7J5cCK#tool={feedback_toolname}" class="button is-link" target="_blank">
+          <a href="https://uaf-iarc.typeform.com/to/mN7J5cCK#tool=HistoricalWinds%20at%20Alaska%20Airports" class="button is-link" target="_blank">
```

Comment:
Add a space between Historical and Winds by using the URLEncoded symbol for space, which is `%20`.  Verify it.